README.rst

```diff
@@ -2,17 +2,16 @@ intel-iot-refkit
 ################
 
 This README file contains information on setting up, building, and booting
-Iot Reference OS Kit for Intel(r) Architecture project.
+IoT Reference OS Kit for Intel(r) Architecture project.
 
 Installing Docker on Linux
 ==========================
 
-It's possible to build the project in a Docker container. Using Docker provides
+We recommend you build projects in a Docker container. Using Docker provides
 a packaged and controlled environment for building an
-image, eliminating development environment issues that
+image and can eliminate development environment issues that
 can occur with differing Linux OS distros and versions, different host
-compilers, and such. (There are instructions later below for building
-without Docker.)
+compilers, and such. (We also describe `building without docker`_ in the instructions below.)
 
 Instructions for installing Docker for common Linux distros on your
 development machine (including Fedora and Ubuntu) are available at:
```

Comment:
Do we support non-Linux development systems (e.g. MacOS/Darwin or Windows)? If not, I would suggest making this clear here.